### PR TITLE
fixed ColTagList to allow tags in to be serchable in list

### PR DESCRIPTION
**Pull request by Bobby-Tav**
## Description
I refactored the code in ColTagList to be allow the tags to searched again.

[Video](https://www.loom.com/share/f3f1ca2c71bd43cb95701410fdd587e2)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by stu-wd — APPROVED** [review]
Thanks for doing this. The video explains the feature precisely.

File: src/components/common/ColTagList.js [hunk 1]
```diff
@@ -14,17 +14,23 @@ function ColTagList(props) {
         className="innerTag"
         data-testid="doc-tag"
         onClick={() => {
-          searchDocs(tag, authState);
+          searchDocs(tag, authState, 1, pageSize);
+          setCurrentSearch(tag, 1, pageSize);
         }}
       >
-        {tag.length < 45 ? (
+        {tag.length < 36 ? (
```
Comment (allisonhomem) [inline src/components/common/ColTagList.js:21]: This change wasn't covered in the video... is this part of the bug fix? It looks like you are slicing the tags to get rid of the overhang? Just curious how this looks on the page.

**Review by allisonhomem — DISMISSED** [review]
It looks like you successfully resolved the issue! I think there might be code that was changed that was not covered in the video, and I'm not sure if it is relevant to this bug fix (check my comment on line 21 of the code).

File: src/components/common/ColTagList.js (hunk 1 above)
Comment (Bobby-Tav) [reply src/components/common/ColTagList.js:21]: I was actually playing with the code and i guess i forgot to change that part back, that is a good catch , i'll fix it right away. Thanks

**Review by mgreenwaldm — APPROVED** [review]

**Review by MMostella — APPROVED** [review]
Great explanation!

**Review by SomeiLam — APPROVED** [review]
The code looks clean and the video is clear.

**Pull request merged**